docs/deployment.md

```diff
@@ -56,7 +56,7 @@ Glitch lets you host node applications for free and edit them directly in your b
    ```
    APP_ID=<your app id>
    WEBHOOK_SECRET=<your app secret>
-   PRIVATE_KEY_PATH=.data/private-key.pem
+   PRIVATE_KEY_PATH=./data/private-key.pem
```

Comment:
I think this originally came from the usage with Glitch. Glitch does not show the `.data/` folder to users who are invited as editors to an app, so it is save to store credentials there. 

If we want to change the default path shown here, then I’d change it to `./private-key.pem`

Reply:
I got it. Thanks a lot